### PR TITLE
Fix regressions

### PR DESCRIPTION
**Pull request by sdimitro**
pylint: mainly caused by drgn signature changes
mypy: caused by drgn signature and behavior change
pytest: commit 92fc92bcd1f66a371aae37e159a9ebdaf6e9f0e2 regression

---

I've been getting notifications of the test failures to my personal email everyday so I figured I'd fix failures :P

### CODE REVIEW

**Comment by sdimitro** [comment]
@jwk404 @prakashsurya @pcd1193182 @palash-gandhi @sebroy - for some reason I cannot add reviewers here.
Feel free to merge this if you want.

I've also created a "hard" fork of this repo (i.e. did not use the Github fork button) so I'll be probably silencing notifications from the Delphix repo soon. If it ends up being easier for your maintenance feel free to point linux-pkg to build directly from my new repo.

**Review by sebroy — APPROVED** [review]

**Review by jwk404 — APPROVED** [review]

**Review by palash-gandhi — APPROVED** [review]

**Comment by sdimitro** [comment]
Can someone merge this for me? :) Need write access to merge

**Pull request merged**
